modules/openshift-developer-cli-odo-getting-support.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/odo-release-notes.adoc
+
+[id="odo-getting-support_{context}"]
+= Getting Support
+
+.Documentation:
+
+If you have found an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose *{product-title}* product type and *Documentation* component type.
+
+.Product:
+
+If you have found an error, encountered a bug or have suggestions for improving the functionality of {odo-title}, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose *{product-title}* product type and *{odo-title}* component type.
+
+Provide as many details in the description of the issue as possible.
+
+.Getting in touch with the product team:
+
+Follow the news in {odo-title} link: https://groups.google.com/forum/#!forum/odo-users[Google Group].
```

Comment:
Suggest either putting these two things in bullets, or combining them into a single sentence like:

```
Follow news in the {odo-title} link:https://groups.google.com/forum/#!forum/odo-users[Google Group] or subscribe to the `odo-users@googlegroups.com` mailing list.
```

(Also note that you need to remove the space between "link:" and "https://...")

Reply:
@boczkowska @bergerhoffer 

I am uncomfortable about this link here. Is the discussion in this group RH supported? If not, we should remove it, or conditionalize this for OKD (upstream) only.

@sferich888 if you have any thoughts here.

Reply:
@vikram-redhat You're right, it's intended for upstream and is not RH supported. I'm removing it.